workspaces/sample-minimal-workspace/client-extensions/dog-batch/build.gradle

```diff
@@ -0,0 +1,7 @@
+dockerDeploy {
```

Comment:
Maybe we could comment here that this will only be needed until https://issues.liferay.com/browse/LPS-172471 is implemented?